### PR TITLE
[NCL-7215] Implement GroupConfigDetailsPage

### PR DESCRIPTION
**Pull request by DnsZhou**

### CODE REVIEW

File: src/services/groupConfigApi.ts [hunk 1]
```diff
@@ -35,3 +41,48 @@ export const build = (
...
+}
+
+/**
+ * Get all Build Configs of a Group Config with the latest Build.
+ *
+ * @param groupConfigId - The id of the Group Config
+ *
+ * @param requestConfig - Axios based request config
+ */
+export const getBuildConfigsWithLatestBuildByGroupConfigId = (
```
Comment (patrikk0123) [inline src/services/groupConfigApi.ts:76]: `getBuildConfigsWithLatestBuild`

**Comment by DnsZhou** [comment]
Hi @matedo1 , all changes mentioned in jira have been finished, and feel free to review the code please.

**Comment by DnsZhou** [comment]
Hi @patrikk0123, the old grid system has been removed, and feel free to implement the same layout with your version of the side bar. Thanks!

**Comment by DnsZhou** [comment]
closed as replaced by https://github.com/project-ncl/pnc-web-ui-react/pull/303

**Pull request closed without merging**